src/main/java/org/apache/flink/benchmark/UnalignedCheckpointTimeBenchmark.java

```diff
@@ -42,17 +43,22 @@
 
 import static java.util.concurrent.TimeUnit.HOURS;
 
+/**
+ * The benchmark for measuring the time taken to finish the configured number of
+ * unaligned checkpoints.
+ */
 @OutputTimeUnit(HOURS)
```

Comment:
NIT: maybe per minute?

Reply:
Make sense, already adjusted to `MINUTE`.